templates/web/oxfordshire/index-steps.html

```diff
@@ -1,3 +1,9 @@
+<h2>[% loc('What we can and can’t fix') %]</h2>
+
+<div class="embed-responsive embed-responsive-16by9">
+    <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/2mxbgJXGHTQ" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
On the [Freshdesk ticket](https://mysocietysupport.freshdesk.com/a/tickets/1648) I asked Oxfordshire if they were going to put the video on an official account, as this video is currently on Tom's personal account. Their response was "I'm happy to use whatever works best for you guys. YouTube, Vimeo? or just hosted on a server somewhere. Happy for your technical needs to dictate where it's held.".

So I think we need to send Oxfordshire a message via the Freshdesk thread asking them to uploads the videos to an official Oxfordshire County Council YouTube channel, then once that's done we can update these embeds to point at the new videos.

Reply:
Cool, I'll ask @lizettal for an account, because I don't have one on fresh desk.

Reply:
Great, Lizetta will send a message to the council.

Reply:
@jacquelinebylau can you please follow up with the council on this as just mentioned

Reply:
@lucascumsille 
Here are the links:
Intro: https://youtu.be/Uf0b_JAzpcY
why we haven't fixed: https://youtu.be/hae9g170sA4

Reply:
Thanks @jacquelinebylau. Appreciate it!